### PR TITLE
feat: add terminal block explorers

### PR DESCRIPTION
**Pull request by gsk967**
Add terminal block explorers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,6 +113,12 @@ View Inter-Blockchain Communication (IBC) transfer activity. The map traces IBC
 * [Map of Zones](https://mapofzones.com/?period=24) - [Source](https://github.com/mapofzones)
 * [Mintscan](https://hub.mintscan.io) - Interchain Explorer by Cosmostation.
 
+## Terminal Block Explorers
```
Comment (julienrbrt) [inline README.md:116]:
```suggestion
### Terminal Block Explorers
```

File: README.md [hunk 2]
```diff
@@ -112,6 +113,12 @@ View Inter-Blockchain Communication (IBC) transfer activity. The map traces IBC
...
+
+View the block explorer on terminal without any web application (firefox, chrome...)
+
+* [cshtop](https://github.com/gsk967/cshtop) - Cosmos htop , Blocks visualizer on terminal.
+* [pvtop](https://github.com/blockpane/pvtop) - Consensus visualizer on terminal.
```
Comment (julienrbrt) [inline README.md:121]:
```suggestion
* [pvtop](https://github.com/blockpane/pvtop) - Consensus visualizer on terminal.

```

File: README.md [hunk 3]
```diff
@@ -22,6 +22,7 @@ Please read the [Contributing guide](./CONTRIBUTING.md). Thank you to all our [c
     * [Rust](#rust)
 * [Block Explorers](#block-explorers)
     * [Visual Block Explorers](#visual-block-explorers)
+    * [Block Explorers on Terminal UI](#terminal-block-explorers)
```
Comment (julienrbrt) [inline README.md:25]:
```suggestion
    * [Terminal Block Explorers](#terminal-block-explorers)
```

File: README.md [hunk 4]
```diff
@@ -112,6 +113,12 @@ View Inter-Blockchain Communication (IBC) transfer activity. The map traces IBC
 * [Map of Zones](https://mapofzones.com/?period=24) - [Source](https://github.com/mapofzones)
 * [Mintscan](https://hub.mintscan.io) - Interchain Explorer by Cosmostation.
 
+## Terminal Block Explorers
+
+View the block explorer on terminal without any web application (firefox, chrome...)
+
+* [cshtop](https://github.com/gsk967/cshtop) - Cosmos htop , Blocks visualizer on terminal.
```
Comment (julienrbrt) [inline README.md:120]: Could you add https://github.com/cosmos/gex before this one?

Comment (gsk967) [reply README.md:121]: Ok

**Review by julienrbrt — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -112,6 +113,14 @@ View Inter-Blockchain Communication (IBC) transfer activity. The map traces IBC
 * [Map of Zones](https://mapofzones.com/?period=24) - [Source](https://github.com/mapofzones)
 * [Mintscan](https://hub.mintscan.io) - Interchain Explorer by Cosmostation.
 
+### Terminal Block Explorers
+
+View the block explorer on terminal without any web application (firefox, chrome...)
+
+* [gex](https://github.com/cosmos/gex) - GEX In-Terminal Gaia Explorer.
```
Comment (julienrbrt) [inline README.md:120]:
```suggestion
* [gex](https://github.com/cosmos/gex) - GEX In-Terminal Explorer.
```

File: README.md [hunk 6]
```diff
@@ -112,6 +113,14 @@ View Inter-Blockchain Communication (IBC) transfer activity. The map traces IBC
 * [Map of Zones](https://mapofzones.com/?period=24) - [Source](https://github.com/mapofzones)
 * [Mintscan](https://hub.mintscan.io) - Interchain Explorer by Cosmostation.
 
+### Terminal Block Explorers
+
+View the block explorer on terminal without any web application (firefox, chrome...)
```
Comment (julienrbrt) [inline README.md:118]:
```suggestion
Explore Cosmos SDK blockchains via a terminal.
```

**Pull request merged**